client/src/pages/Profiles/Profiles.tsx

```diff
@@ -0,0 +1,120 @@
+import { Button, TextField, Typography, Box } from '@mui/material';
+import useStyles from './useStyles';
+import hero from '../../images/landing/hero.jpg';
+import { useEffect, useState } from 'react';
+import { getProfiles } from '../../helpers/APICalls/getProfiles';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import ProfileCard from './ProfileCard/ProfileCard';
+import SearchBar from './SearchBar/SearchBar';
+
+export default function Profiles() {
+  const classes = useStyles();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const [profiles, setProfiles] = useState<any[]>([]);
```

Comment:
Please avoid using any as a type, if we have a Profile interface we can use that here instead (if we don't, you can create one)

Ditto where applicable